SharePointFramework/PortfolioWebParts/src/data/index.ts

```diff
@@ -203,19 +204,40 @@ export class DataAdapter {
         siteId,
         siteIdProperty
       )
+      const projectList = await sp.web.lists
```

Comment:
This doesn't scale well (beyond 500 items), and is also hardcoded wrt. "Avsluttet".

Reply:
+1